isis/CMakeLists.txt

```diff
@@ -279,7 +275,7 @@ find_package(Jama      125     REQUIRED)
 find_package(NN                REQUIRED)
 find_package(OpenCV    3.1.0   REQUIRED)
 find_package(PCL               REQUIRED)
-find_package(protobuf          REQUIRED CONFIG)
+find_package(Protobuf          REQUIRED)
```

Comment:
What did removing the CONFIG line accomplish?

Reply:
The version of protobuf was using a FindProtobuf rather than a .config file. I think it could go back and not cause problems but it's not using a config